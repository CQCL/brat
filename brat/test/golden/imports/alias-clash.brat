import lib as A (x, y)
import lib as A (z)

main :: Nat, Int, Bool
main = A.x, A.y, A.z