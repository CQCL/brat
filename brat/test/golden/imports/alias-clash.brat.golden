Error in alias-clash@2:15
import lib as A (z)
              ^

  Name clash: Alias not unique: "A"

